Lib/http/server.py

```diff
@@ -857,6 +862,9 @@ def guess_type(self, path):
         slow) to look inside the data to make a better guess.
 
         """
+        if not mimetypes.inited:
+            mimetypes.init() # try to read system mime.types
+            self.extensions_map.update(mimetypes.types_map)
```

Comment:
This is actually going to update in the wrong order - it will overwrite any manual overrides with the original type.

Reply:
> it will overwrite any manual overrides with the original type

By manual overrides, are you talking about the `'': 'application/octet-stream'` pair?

Maybe we should replace this line with:

```py
self.extensions_map = {
   '': 'application/octet-stream',
   **mimetypes.types_map
}
```

What do you think?